src/index.js

```diff
@@ -32,7 +32,18 @@ const DEFAULTS = {
    * @type {string}
    * @default mongodb://localhost:27017
    */
-  uri: 'mongodb://localhost:27017'
+  uri: 'mongodb://localhost:27017',
+
+  /**
+   * MongoDB Driver options
+   *
+   * @name MongoDBAdapter#mongoDriverOpts
+   * @type {object}
+   * @default { ignoreUndefined: true }
+   */
+  mongoDriverOpts: {
+    ignoreUndefined: true
```

Comment:
@danivek Depends on which doc you're looking at.. they both are accepted.

ref.:
[mongoclient.connect](http://mongodb.github.io/node-mongodb-native/2.1/reference/connecting/connection-settings/)
[mongoclient](https://mongodb.github.io/node-mongodb-native/api-generated/mongoclient.html)